Add brackets.scm to enable highlighting.

mtalexan opened the pull request:
Add the brackets.scm file to do highlighting of double quotes, parentheses, and curly braces.

/closes #17

caius commented on extension.toml:
```diff
@@ -1,7 +1,7 @@
 name = "Make"
 description = "Makefile syntax highlighting"
-version = "1.1.0"
+version = "1.1.1"
```
Can you drop this out of the PR please, I bump versions separately to feature work 😄

mtalexan commented:
I rebased these changes onto the latest `main`, dropped the change to the version number, and updated the Changelog.  I also retroactively added the Chagelog entry for the `foreach` change that was already merged.

caius reviewed (approved):
Awesome, thanks for the contribution
